Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Adding arguments
-ARG TAG=local
+ARG TAG=edit-azure-flow
```

Comment:
Should this go back to `local` now that we intend to merge with main?

Reply:
```suggestion
ARG TAG=local
```